src/components/Disputes/DisputeCard.js

```diff
@@ -39,32 +35,38 @@ function DisputeCard({ dispute, onSelectDispute }) {
         >
           <strong> Dispute #{id}</strong>
         </h3>
-        <DisputeText
-          text={description}
-          css={`
-            overflow: hidden;
-            ${textStyle('body2')};
-            color: ${theme.content};
-            line-height: ${27}px; // 27px = line-height of textstyle('body1')
-            height: ${27 * 3}px; // 27px * 3 = line-height * 3 lines
-            display: -webkit-box;
-            -webkit-box-orient: vertical;
-            -webkit-line-clamp: 3;
-          `}
-        />
+        {dispute.status !== Status.Voided && (
+          <>
+            <DisputeText
+              text={description}
```

Comment:
Wondering if we should still communicate something here, since still have the card.